### ISSUE TITLE
blenderseed doesn't unregister correctly

### ISSUE THREAD

**Issue by jdent02** [appleseedhq/blenderseed#177]
disabling and re-enabling the addon causes 'already registered as subclass' errors on multiple classes.

**Comment by jdent02** [comment]
This might have been a false alarm.  It uninstalls properly but reinstalling it in the same session causes errors as the bin path gets erased so none of the OSL stuff initializes.

**Comment by dictoon** [comment]
I could reproduce this issue. This PR makes registering/unregistering blenderseed rock-solid from a user's point of view: https://github.com/appleseedhq/blenderseed/pull/249

We do however emit a number of error messages:

```
...
[appleseed] Registering class <class 'blenderseed.ui.lamps.AppleseedLampPanel'>...
...
[appleseed] Unregistering class <class 'blenderseed.ui.lamps.AppleseedLampPanel'>...
[appleseed] ERROR: Failed to unregister class <class 'blenderseed.ui.lamps.AppleseedLampPanel'>: unregister_class(...):, missing bl_rna attribute from 'RNAMeta' instance (may not be registered)
...
```

**Comment by dictoon** [comment]
Fixed in #249.

**Comment by jdent02** [comment]
Should be fixed in 2.8
